### PR TITLE
(test) O3-2271: Write the search by demographics E2E test

### PR DESCRIPTION
**Pull request by anjula-sack**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

- In this test, search by demographics feature is being tested, A simple demographics search query is being made and check for the success notification.

## Screenshots
image

## Related Issue

https://issues.openmrs.org/browse/O3-2271

## Other

None

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -7,14 +7,15 @@ const config: PlaywrightTestConfig = {
...
+    timeout: 80 * 1000,
   },
   fullyParallel: true,
+  outputDir: "../test-results/results",
   forbidOnly: !!process.env.CI,
   retries: 0,
   reporter: process.env.CI
     ? [["junit", { outputFile: "results.xml" }], ["html"]]
-    : [["html"]],
+    : [["html", { outputFolder: "../test-results/report" }]],
```
Comment (anjula-sack) [inline playwright.config.ts:18]: Until the webpack watching issue is resolve we can use this.

File: e2e/specs/demographics.spec.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    await page.getByRole("tab", { name: "Alive" }).click();
+  });
+
+  await test.step("And I click the search button", async () => {
+    await cohortBuilderPage.searchButton().click();
+  });
+
+  await test.step("Then should get a success notification", async () => {
+    await expect(cohortBuilderPage.successNotification()).toBeVisible();
+  });
```
Comment (jayasanka-sack) [inline e2e/specs/demographics.spec.ts:42]: Can we also check for the result?

File: playwright.config.ts (hunk 1 above)
Comment (jayasanka-sack) [reply playwright.config.ts:18]: Awesome. We need to edit the action file too. Also update line 16 too.

File: playwright.config.ts [hunk 3]
```diff
@@ -7,14 +7,15 @@ const config: PlaywrightTestConfig = {
...
   expect: {
-    timeout: 20 * 1000,
+    timeout: 40 * 1000,
   },
   fullyParallel: true,
+  outputDir: "../test-results/results",
   forbidOnly: !!process.env.CI,
   retries: 0,
   reporter: process.env.CI
     ? [["junit", { outputFile: "results.xml" }], ["html"]]
```
Comment (jayasanka-sack) [inline playwright.config.ts:17]:
```suggestion
    ? [["junit", { outputFile: "results.xml" }], ["HTML", { outputFolder: "../test-results/report" }]]
```

File: playwright.config.ts (hunk 1 above)
Comment (anjula-sack) [reply playwright.config.ts:21]: github action?

**Review by jayasanka-sack — APPROVED** [review]
Thanks, @anjula-sack!

**Pull request merged**